src/commands/init.ts

```diff
@@ -31,6 +31,69 @@ export default {
           message.channel.send('Database initialized successfully!');
           console.log('Database tables initialized.');
         }
+        if (type == 'commands') {
+          if (message.author.id != botOwnerId) {
```

Comment:
// Initialize server variables before using this argument!  moderatorID returns "undefined" if not initialized and therefore sends an error to the output chain.